src/components/data-testid/dataset-page.cy.tsx

```diff
@@ -0,0 +1,5 @@
+export const FilterDatasetDataCy = {
```

Comment:
We discussed that filename should be in the format <component_name> + "cy.ts". Object also should have the component-like name. Please fix that.
Same for all cy-files.